server/src/db/migrations/20210202143725_addVotedToVotes.cjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.table("votes", (t) => {
+    t.boolean("voted").notNullable()
```

Comment:
Curious what this column does for us? If a Vote maintains info on who created the vote and for which trip, why do we need a boolean to check if they have voted? We should be able to query based on the userId and tripId to see if there is already a record